doc/nest-10.md

```diff
@@ -0,0 +1,14 @@
+# Applying the transactions to your dependency graph
+
+`nest merge` is a two-step sub-command. To be able to use it, you'll need to have modified your current dependecy graph in some way. For example by having added requirements or removed some. As explained before, Nest will generate transactions for your current dependency graph. This is where `nest merge` comes in play.
+
+This command will apply all the transactions generated by the previous command(s) to go from your current dependency graph to the wanted dependency graph. This is really important to not forget to use `nest merge` after you're done with all the transactions, otherwise they will never be applied.
```

Comment:
Sub commands were not generated before the call to `nest merge`, they are now!

Nest merge first compare the dependency graph, extract a list of transactions, and apply them in order.